cylc/rose/utilities.py

```diff
@@ -25,6 +25,7 @@
 
 from cylc.flow.hostuserutil import get_host
 from cylc.flow import LOG
+from cylc.flow.flags import cylc7_back_compat
```

Comment:
(In practice this works. However it does make a copy of the flag instead of using the flag directly, so I would be tempted to instead do `import cylc.flow.flags.cylc7_back_compat`)

Reply:
I'm not sure that it matters, except in terms of using a trivially small amount of extra memory?

Reply:
In principle a flag should be used directly in case its value changes elsewhere, I think. But doesn't affect this particular flag as it shouldn't change once set to `True`